examples/next/faustwp-getting-started/.eslintignore

```diff
@@ -0,0 +1,2 @@
+components
+pages
```

Comment:
These were temporarily added due to a slew of linting errors within the faustwp example project.

Will need to follow up here and address them all.